src/website/views/components/picker-group.pug

```diff
@@ -31,6 +31,75 @@ p.-text To render a picker group, wrap a series of pickers in a div and apply th
       </label>
     </div>
 
+h3 Texts and icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom"&gt;&lt;/i&gt;</code>.
+.example.-mb--3
+  .-p--3
+    .m-pickerGroup
+      input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+      label(for=`ib-radio1`)
+        i.a-icon.icon-atom
+        span
+          = `Radio1`
+      input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+      label(for=`ib-radio2`)
+        span
+          = `Radio2`
+        i.a-icon.icon-atom
+      input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+      label(for=`ib-radio3`)
+        i.a-icon.icon-atom
+        span
+          = `Radio3`
+        i.a-icon.icon-atom
+
+  :code(lang='html')
+    <div class="m-pickerGroup">
+      <input class="a-picker" type="radio" name="radios" id="radio1">
+      <label for="radio1">
+        <i class="a-icon icon-atom"></i>
+        <span>Radio1</span>
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio2">
+      <label for="radio2">
+        <span>Radio2</span>
+        <i class="a-icon icon-atom"></i>
+      </label>
+      <input class="a-picker" type="radio" name="radios" id="radio3">
+      <label for="radio3">
+        <i class="a-icon icon-atom"></i>
+        <span>Radio3</span>
+        <i class="a-icon icon-atom"></i>
+      </label>
+    </div>
+
+
+h3 Icons
+p.-text Rendering picker group elements with icons <code>&lt;i class="a-icon icon-atom">&lt;/i></code>.
```

Comment:
@janibolkvadze please remove this description as I think it is not necessary. @mattnickles thoughts?